djoser/webauthn/views.py

```diff
@@ -95,6 +97,7 @@ def post(self, request, ukey):
 
 class LoginRequestView(APIView):
     permission_classes = (AllowAny,)
+    serializer_class = Serializer
```

Comment:
Same as the above, shouldn't it be `WebauthnLoginSerializer`?

Reply:
Fixed